languages/en.ini

```diff
@@ -832,10 +832,21 @@ No pickup locations available = "No pickup locations available"
 No Preference = "No Preference"
 No reviews were found for this record = "No reviews were found for this record"
 No Tags = "No Tags"
+no_change_password_scope = "You are not entitled to change password."
+no_delete_notifications_scope = "You are not entitled to delete notifications."
 no_description = "Description not available."
 no_email_address = "Email address missing."
 no_items_selected = "No Items were Selected"
 no_proxied_user = "No proxied user (request for yourself)"
+no_read_fees_scope = "You are not entitled to read fees."
```

Comment:
I would recommend moving these messages into the ILSMessages text domain, since I think that's the most appropriate location for these types of messages. This should just be a matter of moving the lines into the appropriate subdirectory of the languages directory, and adding the `ILSMessages::` prefix to the strings when referred to in the code.

I also wonder if the language should be made more consistent. Is there a reason some use "not entitled" while others use "not allowed"? Some of the existing strings simply use "cannot".